### ISSUE TITLE
DAISY audio-only lacks page number navigation (goto page)

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#1311]
...this is because the DTBOOK-to-XHTML conversion actually starts from SMIL (not DTBOOK) with `class="pagenum"` attributes, but no obvious code path to extract `epub:type="pagebreak"` and the corresponding text from NCX PageList.
